IPython/core/interactiveshell.py

```diff
@@ -1621,7 +1626,7 @@ def _inspect(self, meth, oname, namespaces=None, **kw):
         This function is meant to be called by pdef, pdoc & friends.
         """
         info = self._object_find(oname, namespaces)
-        docformat = sphinxify if self.sphinxify_docstring else None
+        docformat = sphinxify(self.object_inspect(oname)) if self.sphinxify_docstring else None
```

Comment:
```suggestion
        docformat = (
            sphinxify(self.object_inspect(oname)) if self.sphinxify_docstring else None
        )
```

Also, probably line length here